src/app/src/components/Home/Header.tsx

```diff
@@ -0,0 +1,191 @@
+  display: 'flex',
+  flexDirection: 'column',
+  gap: 8,
+
+  [theme.breakpoints.up(660)]: {
+    display: 'none',
+  },
+}))
+
+const HomeHeader = () => {
+  const today = dayjs()
+  const [selectedDate, setSelectedDate] = useState(dayjs())
+
+  const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null)
+  const open = Boolean(anchorEl)
+
+  const handleClick = (event: React.MouseEvent<HTMLButtonElement>) => {
+    setAnchorEl(event.currentTarget)
+  }
+  const handleClose = () => {
+    setAnchorEl(null)
+  }
+
+  const formatDate = () => {
+    return selectedDate.format('ddd, MMMM D')
+  }
+
+  const increaseDate = () => {
+    const newDate = selectedDate.add(1, 'day')
+    setSelectedDate(newDate)
+  }
+
+  const decreaseDate = () => {
+    const newDate = selectedDate.subtract(1, 'day')
+    setSelectedDate(newDate)
+  }
+
+  return (
+    <>
+      <Header>
+        <LeftWrapper>
+          <Logo />
+          <Box display="flex" gap={3} alignItems="center">
+            <Button
+              variant="contained"
+              color="inherit"
+              sx={{ height: 40 }}
+              disabled={today.isSame(selectedDate, 'day')}
+              onClick={() => setSelectedDate(today)}
+            >
+              Today
+            </Button>
+            <Box display="flex" gap={1}>
+              <Button
+                variant="outlined"
+                color="inherit"
+                sx={{ width: 40, height: 40, minWidth: 0 }}
+                onClick={decreaseDate}
+              >
+                <ChevronLeft />
+              </Button>
+              <Button
+                variant="outlined"
+                color="inherit"
+                sx={{ width: 40, height: 40, minWidth: 0 }}
+                disabled={today.isSame(selectedDate, 'day')}
+                onClick={increaseDate}
+              >
+                <ChevronRight />
+              </Button>
+            </Box>
+            <Typography variant="body2" fontWeight={700}>
+              {formatDate()}
+            </Typography>
+          </Box>
+        </LeftWrapper>
+        <Box display="flex" alignItems="center" gap={5}>
+          <StatsWrapper>
+            <Box display="flex" flexDirection="column">
+              <Typography variant="body2">Today's Points</Typography>
+              <Typography variant="body2" fontWeight={700}>
+                30
+              </Typography>
+            </Box>
+            <Box display="flex" flexDirection="column">
+              <Typography variant="body2">Total Points</Typography>
+              <Typography variant="body2" fontWeight={700}>
+                3000
+              </Typography>
+            </Box>
+          </StatsWrapper>
+          <IconButton onClick={handleClick}>
+            <Avatar
+              alt="Person"
+              src="https://upload.wikimedia.org/wikipedia/commons/thumb/a/a0/Andrzej_Person_Kancelaria_Senatu.jpg/300px-Andrzej_Person_Kancelaria_Senatu.jpg"
```

Comment:
Hotlinking to wikimedia may be a bad idea, since it creates traceable traffic to us.

Reply:
Just removing src is fine right now since it's a stand in anyway. @danzjamz don't worry about finding a replacement for it